src/main/java/tds/exam/results/messaging/ExamCompletedMessageListener.java

```diff
@@ -37,6 +44,10 @@ public void handleMessage(final String examId) {
         LOG.debug("Received completed exam notification for id: {}", examId);
         final TDSReport report = examResultsService.findExamResults(UUID.fromString(examId));
 
-        //TODO ship the report somewhere
+        try {
+            testIntegrationSystemService.sendResults(UUID.fromString(examId), report);
```

Comment:
I think this should have a single service.  That single service should take the exam id. 

```
testIntegrationSystemService.sendResults(UUID.fromString(examId);
```

```
class TestIntegrationSystemService {
  private final ExamResultsService examResultsService;
  private final TestIntegrationSystemRepository;

  sendResults(UUID examId) {
    //fetch results
   //call repository
  }
}
```